Test failure: Shell commands should give preference to executables in Python Environment

https://github.com/microsoft/vscode-jupyter/actions/runs/4528055141/jobs/7974482853

```
AssertionError: First entry in PATH (/opt/hostedtoolcache/Python/3.10.10/x64/bin) does not point to executable (/opt/hostedtoolcache/python/3.10.10/x64/bin/python)
```

--- a/src/test/datascience/notebook/executionService.vscode.test.ts
+++ b/src/test/datascience/notebook/executionService.vscode.test.ts
@@ -435,18 +435,21 @@ suite('Kernel Execution @kernelCore', function () {
 
         const pathOutput = getCellOutputs(cell3);
         const execOutput = getCellOutputs(cell4);
-        const pathValue = pathOutput.split(path.delimiter);
-        const sysExecutable = execOutput.trim().toLowerCase();
+        const pathValue = pathOutput.split(path.delimiter)[0];
+        const sysExecutable = execOutput.trim();
 
         // First path in PATH must be the directory where executable is located.
-        assert.ok(
-            areInterpreterPathsSame(
-                Uri.file(path.dirname(getNormalizedInterpreterPath(Uri.file(sysExecutable)).fsPath)),
-                Uri.file(pathValue[0]),
-                getOSType(),
-                true
-            ),
-            `First entry in PATH (${pathValue[0]}) does not point to executable (${sysExecutable}). Path Output ${pathOutput} and Exec Output ${execOutput}`
+        const normalizedInterpreterPath = getNormalizedInterpreterPath(Uri.file(sysExecutable)).fsPath;
+        const normalizedBinPath = path.dirname(normalizedInterpreterPath);
+        assert.equal(
+            getOSType() === OSType.Windows ? normalizedBinPath.toLowerCase() : normalizedBinPath,
+            getOSType() === OSType.Windows ? pathValue.toLowerCase() : pathValue,
+            [
+                `First entry in PATH (${pathValue}) does not point to executable (${sysExecutable}).`,
+                `Normalized Interpreter Path = '${normalizedInterpreterPath}'`,
+                `Normalized Bin Path = '${normalizedBinPath}'`,
+                `Path Output ${pathOutput} and Exec Output ${execOutput}`
+            ].join(',')
         );
     });
     test('!python should point to the Environment', async function () {

